README.rst

```diff
@@ -294,6 +294,27 @@ The method returns a StreamList object that contains a list of all the streams
   print stream.name #stream name
   print stream.layoutClassList #['full']
 
+You can change the layout classes for streams in a session by calling the `set_stream_class_list(session_id, stream_list)` method of the `OpenTok` class.
```

Comment:
Should this be `set_stream_class_lists` because there are multiple? What do you think?

Reply:
Yes, I think is fair to use the plural because there are multiple